okapi-core/src/main/java/org/folio/okapi/managers/TimerManager.java

```diff
@@ -0,0 +1,344 @@
+package org.folio.okapi.managers;
+
+import io.vertx.core.Future;
+import io.vertx.core.MultiMap;
+import io.vertx.core.Vertx;
+import io.vertx.core.eventbus.EventBus;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonObject;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.logging.log4j.Logger;
+import org.folio.okapi.bean.InterfaceDescriptor;
+import org.folio.okapi.bean.ModuleDescriptor;
+import org.folio.okapi.bean.ModuleInstance;
+import org.folio.okapi.bean.RoutingEntry;
+import org.folio.okapi.bean.TimerDescriptor;
+import org.folio.okapi.common.ErrorType;
+import org.folio.okapi.common.OkapiLogger;
+import org.folio.okapi.util.LockedTypedMap1;
+import org.folio.okapi.util.OkapiError;
+
+public class TimerManager {
+
+  private final Logger logger = OkapiLogger.get();
+  private static final String TIMER_ENTRY_SEP = "_";
+  private static final String MAP_NAME = "org.folio.okapi.timer.map";
+  private static final String EVENT_NAME = "org.folio.okapi.timer.event";
+  private final Map<String,LockedTypedMap1<TimerDescriptor>> tenantTimers = new HashMap<>();
+  private final Map<String,Long> timerRunning = new HashMap<>();
```

Comment:
Please update README to mention there is no db persistence for timer change.

Reply:
I'm about to add persistense too,

Reply:
Done.